R/table_with_settings.R

```diff
@@ -72,7 +72,7 @@ table_with_settings_ui <- function(id, ...) {
 #'   }
 #' )
 #' if (interactive()) {
-#'   shinyApp(app$ui, app$server)
+#'   app
```

Comment:
Why this change? We just changed `runApp` to `shinyApp` throughout.

Reply:
It was my bad. I overlooked it while making that change. `app` is teal app! There were 3 or 4 such examples. I missed this one.

Reply:
Not a `teal` app but an app it is. I thought it was agreed to use `shinyApp` in all examples. Perhaps I missed something.

Reply:
OK, now I'm confused. If `app` is an app, the example should conclude with running it, should it not? If the point of the example is to _print_ the app, hten the `if (interactive())` is not necessary. Please explain.

Reply:
Oooh, I forgot `init` doesn't actually return an app object :sweat_smile:  Sorry about that, all good.